packages/block-library/src/cover/edit.native.js

```diff
@@ -483,7 +430,15 @@ const Cover = ( {
 
 	return (
 		<View style={ styles.backgroundContainer }>
-			{ controls }
+			<Controls
```

Comment:
Thank you for moving this to its own component, this file was getting pretty long 😅